src/algorithm.rs

```diff
@@ -16,13 +16,17 @@ const PRECOMMIT_BELOW_TWO_THIRDS: i8 = 0;
 const PRECOMMIT_ON_NOTHING: i8 = 1;
 const PRECOMMIT_ON_NIL: i8 = 2;
 const PRECOMMIT_ON_PROPOSAL: i8 = 3;
+#[cfg(feature = "verify_req")]
+const PRECOMMIT_WITHOUT_VERIFY: i8 = 4;
 const TIMEOUT_RETRANSE_COEF: u32 = 15;
+#[cfg(feature = "verify_req")]
+const VERIFY_AWAIT_COEF: u32 = 50;
 const TIMEOUT_LOW_HEIGHT_MESSAGE_COEF: u32 = 300;
 const TIMEOUT_LOW_ROUND_MESSAGE_COEF: u32 = 300;
 
 /// BFT step
 #[derive(Serialize, Deserialize, Debug, PartialEq, PartialOrd, Eq, Clone, Copy, Hash)]
-pub enum Step {
+pub(crate) enum Step {
     /// A step to determine proposer and proposer publish a proposal.
```

Comment:
This #69  should be resolved first. otherwise, external users will not be able to use the library.

Reply:
> This #69 should be resolved first. otherwise, external users will not be able to use the library.

https://github.com/cryptape/bft-rs/blob/develop/src/lib.rs#L50-L57

Reply:
Step should not expose to external users.